kernel/kernel-api/src/main/java/io/delta/kernel/client/FileSystemClient.java

```diff
@@ -0,0 +1,27 @@
+package io.delta.kernel.client;
```

Comment:
Do we want to add license header for all new files?

Reply:
Added good catch